src/components/Coupon/Coupon.test.jsx

```diff
@@ -119,5 +123,71 @@ describe('<Coupon />', () => {
       expect(mockOnCollapse).toBeCalledTimes(1);
       expect(wrapper.find(Coupon).find(CouponDetails).prop('isExpanded')).toBeFalsy();
     });
+
+    it('expands on keydown of collapsed coupon', () => {
+      const mockOnExpand = jest.fn();
+      wrapper = mount((
+        <CouponWrapper
+          onExpand={mockOnExpand}
+        />
+      ));
+
+      simulateCouponKeyDown('Enter');
+      expect(mockOnExpand).toBeCalledTimes(1);
+      expect(wrapper.find(Coupon).find(CouponDetails).prop('isExpanded')).toBeTruthy();
+    });
+
+    it('collapses on keydown of expanded coupon', () => {
+      const mockOnCollapse = jest.fn();
+      wrapper = mount((
+        <CouponWrapper
+          onCollapse={mockOnCollapse}
+        />
+      ));
+
+      simulateCouponKeyDown('Enter');
+      expect(wrapper.find(Coupon).find(CouponDetails).prop('isExpanded')).toBeTruthy();
+
+      simulateCouponKeyDown('Enter');
+      expect(mockOnCollapse).toBeCalledTimes(1);
+      expect(wrapper.find(Coupon).find(CouponDetails).prop('isExpanded')).toBeFalsy();
+    });
+
+    it('does not handle unknown keydown event', () => {
+      const mockOnExpandOrOnCollapse = jest.fn();
```

Comment:
question: should we be asserting things about `mockOnExpandOrOnCollapse` here?

Reply:
Good catch, @christopappas! I meant to remove `mockOnCollapse` and assert that `mockOnExpandOrOnCollapse` is not called instead of `mockOnCollapse`!

Reply:
Fixed!